extensions/helpers/getAndroidEnvData.js

```diff
@@ -9,17 +11,18 @@ const executeRegexOnFiles = (regex, files) => {
 module.exports = async context => {
   const { envinfo, filesystem } = context;
   // Version specified by RN project settings
-  const androidAppGradle = await filesystem.readAsync(
-    "./android/app/build.gradle"
-  );
-  const androidGradle = await filesystem.readAsync("./android/build.gradle");
-  const androidData = await envinfo.getAndroidSDKInfo();
+  const appGradlePath = path.join(process.cwd(), "android", "app", "build.gradle")
+  const gradlePath = path.join(process.cwd(), "android", "build.gradle")
+
+  const androidAppGradle = await filesystem.readAsync(appGradlePath)
+  const androidGradle = await filesystem.readAsync(gradlePath)
+  const androidData = await envinfo.getAndroidSDKInfo()
 
   if (androidAppGradle) {
     return {
       androidAppGradle,
-      availableApiVersions: androidData["API Levels"],
-      availableBuildToolsVersions: androidData["Build Tools"],
+      availableApiVersions: androidData[1]["API Levels"],
+      availableBuildToolsVersions: androidData[1]["Build Tools"],
```

Comment:
the androidData variable also now contains system images and ndkVersion if you want to add those.